apps/src/javalab/Playground.js

```diff
@@ -89,6 +114,10 @@ export default class Playground {
       // can't set background if game is over
       return;
     }
+
+    const backgroundElement = this.getBackgroundElement();
```

Comment:
should we reset the background to have no image and 0 opacity in `reset`?

Reply:
Ah yes good call, I'll add that